### PR TITLE
Add subsection on CoViD-19

### PR DESCRIPTION
**Pull request by SpencerDawkins**
This is based on the post-IETF 107 MOPS meeting where we talked about this. I chased some more references.

### CODE REVIEW

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -274,6 +305,23 @@ The combination of the large volume of "torrents" and the peer-to-peer character
...
 
+##Extremely Unpredictable Usage Profiles
+
+The causes of unpredictable usage described in {{sec-unpredictable}} were more or less the result of human choices, but we were reminded during a post-IETF 107 meeting that humans are not always in control, and forces of nature can cause enormous fluctuations in traffic patterns.
+
+In his talk, Sanjay Mishra {{Mishra}} reported that after the CoViD-19 pandemic broke out in early 2020,
+ 
+- Comcast’s streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30 {{Comcast}},
+- AT&T reported a 28% jump in core network traffic (single day in April, as compared to pre stay-at-home daily average traffic), with video accounting for nearly half of all mobile network traffic, while
+social networking and web browsing remaining the highest percentage (almost a quarter each) of overall mobility traffic {{ATT}}, and
```
Comment (GrumpyOldTroll) [inline draft-ietf-mops-streaming-opcons.md:316]: s/remainining/remained/

File: draft-ietf-mops-streaming-opcons.md [hunk 2]
```diff
@@ -274,6 +305,23 @@ The combination of the large volume of "torrents" and the peer-to-peer character
...
+- Comcast’s streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30 {{Comcast}},
+- AT&T reported a 28% jump in core network traffic (single day in April, as compared to pre stay-at-home daily average traffic), with video accounting for nearly half of all mobile network traffic, while
+social networking and web browsing remaining the highest percentage (almost a quarter each) of overall mobility traffic {{ATT}}, and 
+- Verizon reported similar trends with video traffic up 36% over an average day (pre COVID-19) {{Verizon}}.
+
+We note that other operators saw similar spikes during this time period. Craig Labowitz {{Labovitz}} reported 
+
+- Weekday peak traffic increases over 45%-50% from pre-lockdown levels, 
+- A 30% increase in upstream traffic over their pre-pandemic levels, and
+- A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks, as gaming usage also increased.)
```
Comment (GrumpyOldTroll) [inline draft-ietf-mops-streaming-opcons.md:323]: The parenthetical might need to either get left out or include a reference to explain "gaming-related DDoS attacks", which the primary reference itself doesn't do.  Maybe this?
https://venturebeat.com/2018/05/13/why-the-game-industry-is-still-vulnerable-to-distributed-denial-of-service-attacks/

File: draft-ietf-mops-streaming-opcons.md [hunk 3]
```diff
@@ -26,13 +26,11 @@ author:
     ins: A. Begen
     name: Ali Begen
     org: Networked Media
-    country: Turkey
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:29]: Jake, I don't actually know why, but I can't generate the document with the countries in place. I'd be happy to do that, if I could figure out what I'm doing wrong. 

Master had this:

    ins: J. Holland
    name: Jake Holland
    org: Akamai Technologies, Inc.
    street: 150 Broadway
    city: Cambridge, MA 02144
    country: United States of America
    email: jakeholland.net@gmail.com
 -
    ins: A. Begen
    name: Ali Begen
    org: Networked Media
    country: Turkey
    email: ali.begen@networked.media
 -
    ins: S. Dawkins
    name: Spencer Dawkins
    org: Tencent America LLC
    country: United States of America
    email: spencerdawkins.ietf@gmail.com

and somehow, we got that submitted, but I remember struggling with that last time, and I'm not sure I was the one who generated the submitted version.

File: draft-ietf-mops-streaming-opcons.md (hunk 2 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:323]: I agree, and I added the reference you suggested.

File: draft-ietf-mops-streaming-opcons.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:316]: fixed (thanks!)

File: draft-ietf-mops-streaming-opcons.md [hunk 4]
```diff
@@ -69,6 +67,39 @@ informative:
...
+    title: "Comcast sees network traffic surge amid coronavirus outbreak"
+    author:
+      - org: CNBC
+    target: https://www.cnbc.com/video/2020/03/30/comcast-sees-network-traffic-surge-amid-coronavirus-outbreak.html
+    date: March 30, 2020
+  ATT:
+    title: "Tuesday (March 24, 2020) Network Insights"
+    author:
+      - org: AT&T
+    target: https://about.att.com/pages/COVID-19/updates.html
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:87]: Created https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/issues/24 to track this.

File: draft-ietf-mops-streaming-opcons.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (GrumpyOldTroll) [reply draft-ietf-mops-streaming-opcons.md:29]: Interesting.  What error message do you get?  As a first guess, is there a chance xml2rfc or kramdown-rfc2629 could be out of date?  On catalina mine's not complaining about countries with these versions:

~~~
$ xml2rfc --version
xml2rfc 2.45.2
$ kramdown --version
1.17.0
~~~

**Pull request merged**